azure/references/SkuProfile/id.ftl

```diff
@@ -0,0 +1,167 @@
+                    "Values" : [ "Developer", "Standard", "Premium", "Basic", "Consumption" ]
+                },
+                {
+                    "Names" : "Capacity",
+                    "Type" : NUMBER_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : [ "storage", "s3", "baselinedata" ],
+            "Children" : [
+                {
+                    "Names" : "Kind",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Tier",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Replication",
+                    "Type" : STRING_TYPE
+                }
+
+            ]
+        },
+        {
+            "Names" : "db",
+            "Children" : [
+                {
+                    "Names" : "Name",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Tier",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Capacity",
+                    "Type" : NUMBER_TYPE
+                },
+                {
+                    "Names" : "Size",
+                    "Type" : NUMBER_TYPE
+                },
+                {
+                    "Names" : "Family",
+                    "Type" : STRING_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : "secretstore",
+            "Children" : [
+                {
+                    "Names" : "Name",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Family",
+                    "Type" : STRING_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : "network",
+            "Children" : [
+                {
+                    "Names" : "Name",
+                    "Type" : STRING_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : "lb",
+            "Children" : [
+                {
+                    "Names" : "Name",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Tier",
+                    "Type" : STRING_TYPE
+                },
+                {
+                    "Names" : "Capacity",
+                    "Type" : NUMBER_TYPE
+                }
+            ]
+        },
+        {
+            "Names" : "bastion",
+            "Children" : [
+                {
+                    "Names" : "Profiles",
+                    "Children" :
+                        [
+                            {
+                                "Names" : "Processor",
+                                "Type" : STRING_TYPE
+                            }
+                        ]
+                }
+            ]
+        },
+        {
+            "Names" : "spa",
```

Comment:
Should this be for spa or cdn? I wouldn't think a storage account would need this much

Reply:
Ah yep you're right. This was intended for use with any component using App Service Plans. I'll remove it and re-add it when we make use of that resource.